gcs-fetcher/pkg/common/common.go

```diff
@@ -42,22 +43,47 @@ type ManifestItem struct {
 //
```

Comment:
Should we include generation in the 2 URI examples above?

Reply:
Good idea, done